src/cast_core/src/CrudActor.cpp

```diff
@@ -637,6 +637,9 @@ struct FindOperation : public BaseOperation {
         if (opNode["Options"]) {
```

Comment:
(Not related to this line.)

The set of find command options that we support here seems to be quite arbitrary. At the risk of making you do more work, I think we should expand this patch so that it adds support for a useful subset of find command parameters. At the very least, I would hope that we support filter, projection, sort, skip, limit, and collation. There is a good argument for adding things like batchSize, readConcern, tailable, awaitData, and let to this list as well.

The same comment would apply to `FindOneOperation` class below.

Reply:
Thanks for filing those tickets! Looks like all topics under this thread have been resolved now, clicking the "resolve" button.